### PR TITLE
add r-b64

### PR DESCRIPTION
**Pull request by mfansler**
Adds [CRAN package `b64`](https://cran.r-project.org/package=b64) as `r-b64`. Recipe generated with `conda_r_skeleton_helper`, with adjustments for Rust dependency, UCRT, and URL splitting.

I'd be surprised if the win-64 branch builds - I don't think we have anything with R + Rust for Windows. Let's see...

Needed for https://github.com/conda-forge/r-downloadthis-feedstock/pull/3.

## Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mfansler** [comment]
Conda Forge R Team will maintain.

**Comment by mfansler** [comment]
@mlondschien @xhochy @conda-forge/help-rust anyone with R + Rust experience available to help with this? It's needed for another package, but is a bit beyond me. Thanks 🙏

**Comment by xhochy** [comment]
@mfansler Linux is green, I sadly have no idea about the other two platforms.

**Comment by mfansler** [comment]
@xhochy thanks for spending time with this! ❤️ 

I think for now we can just move it out of SR as linux-only. Technically, the package that requires this is `noarch`, so this would unlock that to keep auto-ticking.

File: recipes/r-b64/meta.yaml [hunk 1]
```diff
@@ -20,7 +22,7 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}          # [not win]
-    - {{ stdlib('c') }}            # [not win]
+    # - {{ stdlib('c') }}            # [not win]
```
[the remark this answers is not in the document]
Comment (xhochy) [reply recipes/r-b64/meta.yaml:26]: Sorry, commented only for local debugging.
```suggestion
    - {{ stdlib('c') }}            # [not win]
```

**Review by xhochy — APPROVED** [review]

**Pull request merged**